src/packages/cli/package.json

```diff
@@ -115,23 +118,25 @@
     "rimraf": "3.0.2",
     "snap-shot-it": "7.9.3",
     "sqlite-async": "1.1.0",
-    "sqlite3": "4.1",
+    "sqlite3": "4.2.0",
+    "string-hash": "^1.1.3",
     "strip-ansi": "6.0.0",
     "strip-indent": "3.0.0",
     "tempy": "0.7.1",
     "terminal-link": "2.1.1",
     "ts-jest": "26.4.0",
     "typescript": "4.0.3",
+    "verror": "^1.10.0",
     "ws": "7.3.1"
   },
   "scripts": {
     "test:sqlite": "jest integration.sqlite --forceExit",
     "test:postgresql": "jest integration.postgresql --forceExit --maxConcurrency=10",
-    "test:mysql": "jest integration.mysql --forceExit --maxConcurrency=5",
-    "test:mariadb": "jest integration.mariadb --forceExit --maxConcurrency=5",
-    "test:commands": "./fixtures/test.sh && jest dotenv- format doctor generate studio && jest introspect --forceExit",
+    "test:mysql": "jest integration.mysql --forceExit --maxConcurrency=10",
+    "test:mariadb": "jest integration.mariadb --forceExit --maxConcurrency=10",
+    "test:commands": "./fixtures/test.sh && jest dotenv- format doctor generate studio && jest --forceExit tests__/introspect",
```

Comment:
That command test suite needs a bit more qualification b/c it otherwise would capture the introspection integration tests.